### ISSUE TITLE
Should purger work with query-frontend?

### ISSUE THREAD

**Issue by r4j4h** [cortexproject/cortex-helm-chart#483]
With Helm Chart 2.1.0

and values:

```
purger:
  enabled: true
```

I notice things seem to work as expected, a new purger deployment is added.

I am writing because I notice query-frontend is listed in https://github.com/cortexproject/cortex-helm-chart/pull/407 but my pods running `quay.io/cortexproject/cortex:v1.14.1`  are crash looping:

```
$ kubectl -n cortex logs cortex-query-frontend-656ffb89c6-ng59r


flag provided but not defined: -purger.enable
Run with -help to get list of available parameters
```

With a minimal test, I can reproduce locally:

```
$ podman run --rm -ti quay.io/cortexproject/cortex:v1.14.1 -target=query-frontend -purger.enabled 
flag provided but not defined: -purger.enabled
Run with -help to get list of available parameters
ip-172-30-42-27:~ jhegman$ 
```

I have not been keeping up with the latest evolutions and just recently started standing up a new instance.
I know that blocks did not support deletion but now that chunks is gone and the helm chart incorporated purger I am under the impression it now works.

Am I missing something obvious? Or is this feature still a work in progress?

(
As a sanity check, it starts up fine without the flag:
```
$ podman run --rm -ti quay.io/cortexproject/cortex:v1.14.1 -target=query-frontend 
level=info ts=2023-10-06T21:26:35.10463411Z caller=main.go:194 msg="Starting Cortex" version="(version=1.14.1, branch=HEAD, revision=984ac41)"
level=info ts=2023-10-06T21:26:35.114992577Z caller=server.go:306 http=[::]:80 grpc=[::]:9095 msg="server listening on addresses"
level=info ts=2023-10-06T21:26:35.119425777Z caller=module_service.go:64 msg=initialising module=server
level=info ts=2023-10-06T21:26:35.119728791Z caller=module_service.go:64 msg=initialising module=query-frontend-tripperware
level=info ts=2023-10-06T21:26:35.120083807Z caller=module_service.go:64 msg=initialising module=query-frontend
level=info ts=2023-10-06T21:26:35.121077852Z caller=cortex.go:422 msg="Cortex started"
^C
```
)

**Comment by friedrichg** [comment]
No, it's not implemented in the helm chart. you are welcome to file PR 😄 

---
These endpoints

https://cortexmetrics.io/docs/api/#tenant-delete-request 
https://cortexmetrics.io/docs/api/#tenant-delete-status

need to be added to https://github.com/cortexproject/cortex-helm-chart/blob/master/templates/nginx/nginx-config.yaml

**Comment by nschad** [comment]
Addressed this #488. Can you try again?
